test/unit/specs/components/Personalization/helper/dom/getParent.spec.js

```diff
@@ -10,5 +10,32 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-// eslint-disable-next-line no-unused-vars
 import getParent from "../../../../../../../src/components/Personalization/helper/dom/getParent";
+import {
+  selectNodes,
+  removeNode,
+  appendNode,
+  createNode
+} from "../../../../../../../src/utils/dom";
+import { getElementById } from "../../../../../../../src/components/Personalization/helper/dom";
+
```

Comment:
Probably this should be rename to `Personalization::helper::dom::getParent`

Reply:
thank you